sentinel-extension/sentinel-annotation-aspectj/src/test/java/com/alibaba/csp/sentinel/annotation/aspectj/integration/service/FooService.java

```diff
@@ -53,6 +53,25 @@ public String fooWithFallback(int i) throws Exception {
         return "Hello for " + i;
     }
 
+    @SentinelResource(value = "apiFooWithFallback", globalFallback = AnnotationGlobalFallback.class,
```

Comment:
If convenient, could you please add relevant example code in the sentinel-demo-annotation-spring-aop module?